model/preference_test.go

```diff
@@ -57,8 +57,9 @@ func TestPreferencePreUpdate(t *testing.T) {
 	preference.PreUpdate()
 
 	var props map[string]string
-	if err := json.NewDecoder(strings.NewReader(preference.Value)).Decode(&props); err != nil {
-		t.Fatal(err)
+	err := json.NewDecoder(strings.NewReader(preference.Value)).Decode(&props)
+	if err != nil {
```

Comment:
if check is not needed. require.Nil will take care of that.

Reply:
ok updated it